frontend/src/components/visualizations/Visualization.tsx

```diff
@@ -94,10 +95,14 @@ export const Visualization = (props: previewProps) => {
 								// if no visualization config exist, guess which widget to use by looking at the mime type of
 								// the raw bytes
 								else {
+									//check if file size is greater than threshold, if it is, then show no visualization
+									if (fileSummary && fileSummary.bytes && fileSummary.bytes >= config["rawDataVisualizationThreshold"]) {
+										console.log("File is greater than threshold");
```

Comment:
Short term i think this works. Long term we would benefit from thinking about backend solution to handle this logic.
Also mayb ewe could have a component that display "file is too big for visualization"?